dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/ajax/CMSMaintenanceAjax.java

```diff
@@ -313,6 +346,25 @@ public String doBackupExport(String action, boolean dataOnly) throws IOException
 			f.mkdirs();
 			deleteTempFiles();
 			String message = "";
+		if(action.equals("createZip")) {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "createZip" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1132)